backend/app/cmd/server.go

```diff
@@ -230,6 +230,12 @@ type NotifyGroup struct {
 		Token   string `long:"token" env:"TOKEN" description:"slack token"`
 		Channel string `long:"chan" env:"CHAN" description:"slack channel"`
 	} `group:"slack" namespace:"slack" env-namespace:"SLACK"`
+	Webhook struct {
+		WebhookURL string        `long:"url" env:"URL" description:"webhook notification URL"`
+		Headers    string        `long:"headers" env:"HEADERS" description:"webhook authentication headers in format --notify.webhook.headers=Header1:Value1,Header2:Value2,..."`
```

Comment:
I'm not sure if combining headers into a string ans splitting it after is such a good idea. Is there a reason not to have Headers as []string ?

btw, this is how i did it in in reproxy https://github.com/umputun/reproxy/blob/master/app/main.go#L36

Reply:
I like `[]string` idea. I brought `splitAtCommas` from reproxy project. Added it to `server.go`. Is that OK?